packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -195,7 +195,15 @@ class DatePickerInput extends React.Component {
 
     const label = this.props.ariaLabel ? this.props.ariaLabel : intl.formatMessage({ id: 'Terra.datePicker.date' });
 
-    this.formatDescriptionId = (useExternalFormatMask === false) ? `terra-date-picker-description-format-${this.uuid}` : '';
+    if (useExternalFormatMask === false && inputAttributes && inputAttributes['aria-describedby']) {
+      this.formatDescriptionId = `terra-date-picker-description-format-${this.uuid} ${inputAttributes['aria-describedby']}`;
+    } else if (useExternalFormatMask === false) {
+      this.formatDescriptionId = `terra-date-picker-description-format-${this.uuid}`;
+    } else if (inputAttributes && inputAttributes['aria-describedby']) {
+      this.formatDescriptionId = inputAttributes['aria-describedby'];
+    } else {
+      this.formatDescriptionId = '';
```

Comment:
Given the change made in DatePickerField where `aria-describedby` will always be provided, it should never get in here. If it ever gets in here, should this be set to `undefined` instead of an empty string so that aria-describedby is not defined instead of setting to an empty string?

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1252/commits/fbcd7ea00aed370b88d57583ddaffc66a755f9f7